packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -229,6 +228,98 @@ describe("IntervalCollection fuzz testing", () => {
 	});
 });
 
+// currently fails
+describe.skip("IntervalCollection offline fuzz testing", () => {
```

Comment:
doesn't seem to me like this does any offline fuzz testing--I'd expect that to exercise the applyStashedOp codepath. I don't think that's the intent--maybe just a rename is in order?